addons/website_payment/static/src/snippets/s_donation/000.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates xml:space="preserve">
+    <t t-name="website_payment.donation.prefilledButtons">
+        <button id="s_donation_btn_1" class="s_donation_btn btn btn-outline-primary btn-lg mb-2 mr-1 o_not_editable" contenteditable="false" data-donation-value="10">10</button>
+        <button id="s_donation_btn_2" class="s_donation_btn btn btn-outline-primary btn-lg mb-2 mr-1 o_not_editable" contenteditable="false" data-donation-value="25">25</button>
+        <button id="s_donation_btn_3" class="s_donation_btn btn btn-outline-primary btn-lg mb-2 mr-1 o_not_editable" contenteditable="false" data-donation-value="50">50</button>
+        <button id="s_donation_btn_4" class="s_donation_btn btn btn-outline-primary btn-lg mb-2 mr-1 o_not_editable" contenteditable="false" data-donation-value="100">100</button>
```

Comment:
> Is it safe to rely on `id` within a snippet which might appear several times on the page ?
> (On drop+save without further configuration, ids 1~4 are kept, and `uniqueId()` is not used to assign new ones)

If someone want really puts 2 snippets Donation in the same page (which would be weird). We search each time for IDs only in `this.$target` so no worries for me

Reply:
> > Would it make sense for them to be `d-none` to avoid the flickering when obtaining the current currency ?
> > (This comment is actually also about `s_donation.xml` and `options.js`'s `cleanForSave()`)
> 
> Will the flickering not be worse if we wait for the currency before displaying the buttons? (or I miss something ?)

Would having the whole snippet `d-none` until the buttons are rendered be nicer ?
It's just a bit strange because the other elements with currencies do not have this problem (shop, cart popup, cart...) but they are rendered server-side with the currency already embedded.